intel-retail/automated-vending
# feat: Migrate ds-controller microservice to EdgeX 3.0

## PR Checklist



## What are you changing?


## Issue this PR will close
close: #134 

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable
Run the following make commands from the ds-controller directory, all should run without erros

- make docker
- make test

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/automated-vending )
